src/test/functional/pages/BuzzPage.ts

```diff
@@ -10,22 +10,29 @@ export class BuzzPage extends BasePage {
     readonly photoImg: Locator
     readonly noPostParagraph: Locator
     readonly buzzPageButton: Locator
+    readonly editPostParagraph: Locator
+    readonly textFieldInPostEdit: Locator
+    readonly removePictureButton: Locator
+    readonly confirmEditedPost: Locator
```

Comment:
I found out that if you use a class that extends another class (and `page` is part of the super) you can use it in field declaration instead of constructor. Since BuzPage extends BasePage you can use it to make it a bit cleaner:
`public readonly confirmEditedPost = this.page.locator('.orangehrm-buzz-post-modal-actions button')`